arelle/plugin/validate/EFM/resources/country-deprecated-concepts.json

```diff
@@ -1,19 +1 @@
-{
-"AN": "2015-08-31",
-"EQ": "2015-08-31",
-"GC": "2016-08-31",
-"GZ": "2015-08-31",
-"IU": "2015-08-31",
-"IW": "2015-08-31",
-"IY": "2015-08-31",
-"JQ": "2015-08-31",
-"PQ": "2015-08-31",
-"RH": "2015-08-31",
-"TP": "2015-08-31",
-"TQ": "2015-08-31",
-"UR": "2015-08-31",
-"WB": "2015-08-31",
-"WQ": "2015-08-31",
-"XX": "2016-08-31",
-"YU": "2015-08-31"
-}
\ No newline at end of file
+{"AN": "2015-08-31", "EQ": "2015-08-31", "GC": "2016-08-31", "GZ": "2015-08-31", "IU": "2015-08-31", "IW": "2015-08-31", "IY": "2015-08-31", "JQ": "2015-08-31", "PQ": "2015-08-31", "RH": "2015-08-31", "TP": "2015-08-31", "TQ": "2015-08-31", "UR": "2015-08-31", "WB": "2015-08-31", "WQ": "2015-08-31", "XX": "2016-08-31", "YU": "2015-08-31", "ensure_ascii": false, "indent": 0}
```

Comment:
Why did this change? It made it harder to read and aded fields that are not needed in `ensure_ascii` and `indent`? This applies to all of the deprecated concept json files.